azure-pipelines.yml

```diff
@@ -0,0 +1,34 @@
+# ASP.NET
```

Comment:
This template is not for ASP.NET Core. We would use dotnet tasks, so we could use linux based agents as well.
There is a separate open issue regarding pipeline setup.

Reply:
True, I just wanted to see how easy it is to setup CI for my branch. It's just using VS & VSTest tasks.